### PR TITLE
Load subject when loading a temlate

### PR DESCRIPTION
**Pull request by AEtherC0r3**
Rebase #1212 and fix template loading

### CODE REVIEW

**Comment by differentreality** [comment]
The subject is currently being loaded only for the registration email. How about loading the subject for the other templates as well (proposal, cfp, etc)?

File: app/views/admin/emails/index.html.haml [hunk 1]
```diff
@@ -20,67 +20,58 @@
...
-                %a.btn.btn-link.control_label.load_template{"data-template"=>"Dear {name},\n\nThank you for Registering for the conference {conference}.\nPlease complete your registration by filling out your travel information.\n\nIf you are unable to attend please unregister online:\n{registrationlink}\n\nFeel free to contact us with any questions or concerns.\nWe look forward to see you there.\n\nBest wishes\n\n{conference} Team",
-                "data-name"=>"email_settings_registration_body"} Load Template
+                %a.btn.btn-link.control_label.load_template{"data-subject-input-id" => "email_settings_registration_subject", "data-subject-text" => "Thank you for registering", "data-body-input-id"=>"email_settings_registration_body", "data-body-text"=>"Dear {name},\n\nThank you for Registering for the conference {conference}.\nPlease complete your registration by filling out your travel information.\n\nIf you are unable to attend please unregister online:\n{registrationlink}\n\nFeel free to contact us with any questions or concerns.\nWe look forward to see you there.\n\nBest wishes\n\n{conference} Team"} Load Template
```
Comment (differentreality) [inline app/views/admin/emails/index.html.haml]: The attributes in the hash could go on separate lines to help with readability.

Comment (differentreality) [inline app/views/admin/emails/index.html.haml]: For styling purposes, we can use single quotes here.

File: app/views/admin/emails/index.html.haml [hunk 2]
```diff
@@ -20,67 +20,84 @@
...
                 = render partial: 'help', locals: {id: 'updated_venue_help', show_event_variables: false}
               #cfp.tab-pane{role: "tabpanel"}
                 = f.input :send_on_program_schedule_public, hint: "This will notify all participants when the dates are updated or when the schedule is made public"
                 = f.input :program_schedule_public_subject, hint: "This subject will used whenever dates are updated or when the schedule is made public"
                 = f.input :program_schedule_public_body, input_html: { rows: 10, cols: 20 }
-                %a.btn.btn-link.control_label.load_template{"data-template"=>"Dear {name},\n\nThe Conference Call for Papers Details of {conference} has changed.The schedule is being made public.\n Link to Schedule {schedule_link} \n\nBest wishes\n\n{conference} Team",
-                "data-name"=>"email_settings_program_schedule_public_body"} Load Template
+                %a.btn.btn-link.control_label.load_template{'data-subject-input-id' => 'email_settings_program_schedule_public_subject',
+                'data-subject-text' => 'The schedule has been released',
```
Comment (differentreality) [inline app/views/admin/emails/index.html.haml]: s/released/announced

We should probably rewrite the body of that template to something like:
```
Dear {name},

The schedule for {conference} has been announced.
Link to Schedule {schedule_link} 

Best wishes
{conference} Team
```

**Comment by ChrisBr** [comment]
@differentreality ping!

**Comment by differentreality** [comment]
:+1:  Nice work @AEtherC0r3 :tada:

**Pull request merged**